code/lib/builder-webpack5/templates/virtualModuleModernEntry.js.handlebars

```diff
@@ -6,19 +6,15 @@ import { createChannel as createWebSocketChannel } from '@storybook/channel-webs
 
 import { importFn } from './{{storiesFilename}}';
 
-const { SERVER_CHANNEL_URL } = global;
-
 const getProjectAnnotations = () =>
   composeConfigs([{{#each previewAnnotations}}require('{{this}}'),{{/each}}]);
 
 const channel = createPostMessageChannel({ page: 'preview' });
 addons.setChannel(channel);
 
-if (SERVER_CHANNEL_URL) {
```

Comment:
At first I was worried that removing this if-statement might cause it to run more often, but it looks like it was always true already, because `getServerChannelUrl` always returns a non-empty string already.